setup.py

```diff
@@ -216,14 +216,14 @@ def run(self):
     cwd = pathlib.Path().absolute()
     build_temp = pathlib.Path("build")
     os.chdir(str(build_temp))
+    self.spawn([self.cmake_cmd, '-DBUILD_TYPE=', '-P', 'cmake_install.cmake'] )
```

Comment:
That will launch `cmake -DBUILD_TYPE= -P cmake_install.cmake`. Should not `-DBUILD_TYPE=` be followed by `Release`?

Reply:
Yes of course